hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/HevmStaticTransactionFactory.java

```diff
@@ -87,9 +81,7 @@ public HederaEvmTransaction fromHapiQuery(@NonNull final Query query) {
     }
 
     private void assertValidCall(@NonNull final ContractCallLocalQuery body) {
-        final var minGasLimit =
-                Math.max(INTRINSIC_GAS_LOWER_BOUND, gasCalculator.transactionIntrinsicGasCost(EMPTY, false));
-        validateTrue(body.gas() >= minGasLimit, INSUFFICIENT_GAS);
+        validateTrue(body.gas() >= 0, CONTRACT_NEGATIVE_GAS);
```

Comment:
Is 0 the correct lower bound for CONTRACT_CALL_LOCAL?

Reply:
Yes. That's what we were using in mono code (see ContractCallLocalAnswer:94). That's also the lower bound that we check for in modularized code during prehandle (see ContractCallLocalHandler:82)